devito/passes/clusters/blocking.py

```diff
@@ -103,6 +118,28 @@ def callback(self, clusters, prefix):
         return processed
 
 
+def preprocess(clusters, options):
+    # Preprocess: heuristic: drop TILABLE from innermost Dimensions to
```

Comment:
this could maybe be turned into a docstring now that preprocess has its own function

Reply:
ok